### PR TITLE
Update cl.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds [Kanade TV](http://kanadetv.live/) again. I don't sure if this channel is music or animation, but again this channel is open: https://www.facebook.com/KdeTelevision/

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Seems that they moved from Twitch to their own server.

**Comment by sguinetti** [comment]
Ok. I changed category for now.

File: channels/cl.m3u [hunk 1]
```diff
@@ -151,3 +151,5 @@ http://cdn.streamingmedia.cl:1935/live/vtv2/playlist.m3u8
 http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 #EXTINF:-1 tvg-id="ZappingMusic.cl" tvg-name="Zapping Music" tvg-country="CL" tvg-language="" tvg-logo="https://i.imgur.com/lHvoKdw.jpg" group-title="Music",Zapping (720p) [Offline]
 https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="KanadeTV.do" tvg-name="Kanade TV" tvg-country="LATAM" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/KdeTelevision/picture?width=300&height=300" group-title="Entertainment",Kanade TV [Not 24/7]
```
Comment (Dum4G) [inline channels/cl.m3u:154]: Is it not 24/7 Anime channel? If so, it falls into Animation category

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/cl.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/cl.m3u:154]: Yeah the program seems to be fully Animation, but OP didn't know about the "Music" category, so maybe it has got some musical programming.

Comment (Dum4G) [reply channels/cl.m3u:154]: If 90% of it's content is anime by design I think that doesn't matter that much

Comment (LaneSh4d0w) [reply channels/cl.m3u:154]: You're right.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**